src/main/resources/assets/crimson/lang/en_us.json

```diff
@@ -21,10 +21,13 @@
   "block.crimson.realgar_ore": "Realgar Ore",
   "block.crimson.vanadinite_ore": "Vanadinite Ore",
   "block.crimson.crimson_thorn": "Crimson Thorn",
+	"block.crimson.potted_crimson_thorn": "Potted Crimson Thorn",
```

Comment:
two spaces in json files

Reply:
Fixed in 9f64c8574f19b61a662f9ecfd49a927533642067.